src/chapters/how-programs-work/replit.rst

```diff
@@ -1,48 +1,48 @@
-Repl.it
-=======
+Replit.com
+==========
 
-`Repl.it <https://repl.it>`__ and `Trinket.io <https://trinket.io>`__ share
+`Replit.com <https://replit.com>`__ and `Trinket.io <https://trinket.io>`__ share
 many of the same features, but the two sites differ in how they present their
 workspaces.
 
-The previous page explored Trinket, so now let's take a look at Repl.it.
+The previous page explored Trinket, so now let's take a look at Replit.com
 
-Create a Repl.it Account
--------------------------
+Create a Replit.com Account
+---------------------------
 
 .. admonition:: Warning
 
-   Before signing up for a repl.it account, be sure your school has approved
+   Before signing up for a replit.com account, be sure your school has approved
    the site for student use!
 
-Create a new student account with `Repl.it <https://repl.it/signup>`__.
+Create a new student account with `Replit.com <https://replit.com/signup>`__.
 
 .. figure:: figures/replit-signup.png
-   :alt: The repl.it sign-up screen.
+   :alt: The Replit.com sign-up screen.
 
 After you sign in, you will see your *Home* screen, which displays your recent
 work and announcements. Since you are just starting out, you won't need to do
 much on this screen.
 
 .. figure:: figures/replit-home.png
-   :alt: Repl.it home screen.
+   :alt: Replit.com home screen.
 
    Home screen
 
-Click on *new repl* to begin a new project. Scroll through the options and
-select *Python*. Next, name your project and click *Create Repl*.
+Click on *new repl (1) or (2)* to begin a new project. Scroll through the options and
+select *Python (3)*. Next, name your project (4) and click *Create Repl (5)*.
```

Comment:
I'm struggling with these lines.  On my first reading, I paired them with the home screen figure.  It took a moment to see that they referenced the next screenshot instead.  Could you rework these a little to make the connections more explicit?

Figure tweaks:
1) The red arrows and labels for 1 and 2 fade into the gray background.  I think we should highlight the new repl button by the username and the blue "+" in the home screen figure.  They will stand out more there.  (FYI - One of my former students requested that I avoid red on black pairings, because he could not distinguish between the two).
2) If we move 1 and 2 to the home page screenshot, then the repit-newrepl figure can include only the relevant dialog box, making it larger and easier to read.  Labels 3 - 5 can go here.

Reply:
So I think I've got this now.  However I highlighted the Python under Create as it would be what I would hit since it automatically fills in the language.  One less step for the students.

Reply:
The new arrangement looks good!